web/src/peer_data.ts

```diff
@@ -69,9 +69,15 @@ export function potential_subscribers(stream_id: number): User[] {
     return people.filter_all_users(is_potential_subscriber);
 }
 
-export function get_subscriber_count(stream_id: number): number {
-    const subscribers = get_user_set(stream_id);
-    return subscribers.size;
+export function get_subscriber_count(stream_id: number, include_bots: boolean = true): number {
+    let subscribers = [...get_user_set(stream_id).keys()];
```

Comment:
This has negated the whole point of the `LazySet` optimization by copying the entire set to an array every time.